db/migrate/20201005065240_add_result_to_keyword.rb

```diff
@@ -0,0 +1,10 @@
+class AddResultToKeyword < ActiveRecord::Migration[6.0]
+  def change
+    add_column :keywords, :status, :string, default: 'processing'
```

Comment:
default `processing` isn't real, should it be `created` or `in queue` 🤔

Reply:
Fixed in ba441144c8110a345f1162715b927493464d3bad